Directory.Build.targets

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
+  <PropertyGroup Condition=" '$(PLibCore)' != 'true' ">
+    <SkipCopyBuildProduct>true</SkipCopyBuildProduct>
+    <ClearOutputDirectory>false</ClearOutputDirectory>
+  </PropertyGroup>
```

Comment:
NOTE: These conditions only apply to projects that use ILRepack, so the condition should match the conditions on the ILRepack target.